src/components/modules/ProjectModule/ProjectDetailsContent/ProjectDetailsContent.tsx

```diff
@@ -233,8 +234,17 @@ class ProjectDetailsContent extends React.Component<Props, State> {
         .map(a => ({ value: a.role.id, label: a.role.name }));
       return roles;
     };
+    const hiddenRoles = configLoader.config.hiddenUserRoles || [];
     const allRoles = this.props.roles
-      .filter(r => r.name !== "key-manager:service-admin")
+      .filter(r => {
+        if (r.name === "key-manager:service-admin") {
```

Comment:
You could've also just added this role to the list of `hiddenUserRoles`. Can you please try that?